src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -687,10 +101,10 @@ int mongoSubCacheItemInsert(const char* tenant, const BSONObj& sub)
   cSubP->throttling            = sub.hasField(CSUB_THROTTLING)?       getIntOrLongFieldAsLong(sub, CSUB_THROTTLING)       : -1;
   cSubP->expirationTime        = sub.hasField(CSUB_EXPIRATION)?       getIntOrLongFieldAsLong(sub, CSUB_EXPIRATION)       : 0;
   cSubP->lastNotificationTime  = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLong(sub, CSUB_LASTNOTIFICATION) : -1;
-  cSubP->count                 = sub.hasField(CSUB_COUNT)?            getIntOrLongFieldAsLong(sub, CSUB_COUNT)            : 0;
+  cSubP->count                 = 0;
```

Comment:
Here is where I set count to ZERO, as this function is called during synch.
This is not 100% and a bit error prone.
I should probably remove the reset of count from here and loop over all subs in the cache as the last thing that is done after sync (in subCacheSync) ...

Opinions?